wd_core.au3

```diff
@@ -157,6 +157,7 @@ Global Const $_WD_ErrorElementInvalid = "invalid argument"
 Global Const $_WD_ErrorElementIntercept = "element click intercepted"
 Global Const $_WD_ErrorElementNotInteract = "element not interactable"
 Global Const $_WD_ErrorWindowNotFound = "no such window"
+Global Const $_WD_ErrorSessionNotCreated = "session not created"
```

Comment:
This had crossed my mind. However, that would generally mean that we would return a completely new error code as well.
Not sure if that's a good thing. Could this lead to someone's existing script breaking?

Reply:
I think it is worth to do this.
Even if this will be Script Breaking Change 
But it may be SBC in some very specific situation when somebody check for this `$_WD_ERROR_Exception` specific error instead just checking `If @error Then `
Even in that case the returned value `$sSession = _WD_CreateSession($sCapabilities)` will be `$sSession==""` so calling any next function will fail too.

But in any case.
Good writeen code will not fail.

Reply:
Please review https://github.com/Danp2/au3WebDriver/pull/428/commits/e20162ffbc71a5565dc86692ec0d2f0f83f80032